test/gcloud/logging/project_test.rb

```diff
@@ -19,4 +19,15 @@
     logging.must_be_kind_of Gcloud::Logging::Project
     logging.project.must_equal project
   end
+
+  it "deletes a log" do
```

Comment:
Similar to how Pub/Sub accepts a topic name or the full topic path. See `Gcloud::Pubsub::Connection#topic_path`.